redisson/src/main/java/org/redisson/connection/SentinelConnectionManager.java

```diff
@@ -408,19 +415,28 @@ public void accept(Object t, Throwable u) {
                         continue;
                     }
                     
-                    String ip = map.get("ip");
+                    String host = map.get("ip");
                     String port = map.get("port");
                     String flags = map.getOrDefault("flags", "");
                     String masterLinkStatus = map.getOrDefault("master-link-status", "");
                     String masterHost = map.get("master-host");
                     String masterPort = map.get("master-port");
 
+                    final String ip = convertToIpAddress(NodeType.SLAVE, host, port);
```

Comment:
blocking call can't be used in async callbacks

Reply:
I'm calling syncUninterruptibly on the future, so it should not be async anymore.  I haven't tested this particular change (my previous iteration was just using InetAddress.getByName, but I realized that you had some logic in the RedisClient to get the IP address, and figured we should keep the logic the same).

Reply:
> I'm calling syncUninterruptibly on the future, so it should not be async anymore

It became blocking (synchronous).

Reply:
Are you suggesting that I should have convertToIpAddress return a Future and have a callback here?